hhs_oauth_server/settings/test.py

```diff
@@ -2,6 +2,13 @@
 
 del LOGGING['loggers']
 
+FHIR_SERVER = {
+    "FHIR_URL": "https://fhir.backend.bluebutton.hhsdevcloud.us/v1/fhir/",
+    "CERT_FILE": "ca.cert.pem",
+    "KEY_FILE": "ca.key.nocrypt.pem",
+    "CLIENT_AUTH": True,
+}
```

Comment:
When running tests and having these related ENV variables set I was receiving several failures.

Reply:
why does configuring the values here fix the errors? Could the failures have been a warning sign of brittle configuration logic?

Reply:
I removed this. I think you are right that it doesn't belong here.